hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestBlockOutputStream.java

```diff
@@ -662,7 +661,6 @@ void testWriteExactlyMaxFlushSize(boolean flushDelay, boolean enablePiggybacking
 
   @ParameterizedTest
   @MethodSource("clientParameters")
-  @Flaky("HDDS-11325")
```

Comment:
I don't think `@Flaky` should be removed without other code changes (test or production) that could fix the failure.

Reply:
Yes....it was fixed in a prior commit https://github.com/apache/ozone/commit/2d3ba800ead113b2c789d3643f96abec6ad57b7d

sorry it was confusing.